platform-coop-toolkit/coop-library-framework
# Add required labels (resolves #104)

## Description

Resolves #104, adding `(Required)` labels to required fields. Depends on #109.

## Steps to test

1. Try to submit a resource with required fields empty.

**Expected behavior:** Validation errors are shown. Required fields are clearly labelled.

## Additional information

Not applicable.

## Related issues

- Resolves #104
